R/ivreg.pred_se.R

```diff
@@ -0,0 +1,52 @@
+# Main wrapper function to calculate margins and se
+pred_se.ivreg <- function(df_levels, model, type, base_rn, vcov_mat, weights){
+
```

Comment:
Not quite. When calculating mean values it's not necessary to apply any link function (because we know it's gaussian), and the `deriv_func` function in the `calc_jacob` call is hard-coded since it's not in the model object (and doesn't vary).